prdoc/pr_8250.prdoc

```diff
@@ -0,0 +1,13 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: "Simplify umbrella version detection in CI"
+
+doc:
+  - audience: CI/CD
+    description: |
+      Simplified the umbrella crate version detection in checks-quick.yml to use cargo metadata 
+      with a simple fallback.
+
+crates:
+  # No crates are affected as this only modifies CI workflow files
```

Comment:
We can remove this, CI changes are not really relevant for prdocs, which we write only for crates changes that are user facing.